### DISCUSSION TITLE
New feature: RTL support in translation interface

### DISCUSSION THREAD

**Discussion by ramonvanbezouw** [directus/directus#14664] [Feature Requests]
Hello everyone,

First of all, it's been a delight using Directus so far!

I think it would be a nice addition for the project to also support right to left languages for translations. This would allow us to also use languages like Arabic and Hebrew for translatable fields and make the CMS suitable for a broader range of projects.

To start this discussion @martijn-dev and I set up a quick draft in a pull request. It's far from perfect yet, but gives a rough idea on how this could be implemented.

Pull request: #14665 

A few points to discuss:
- Either set the rtl direction on all fields separately or just once on the parent form (in the current PR we've only added it to the input interface.
- We've currently hardcoded the `direction` property on the `languages` collection. Since this collection can be changed by the user, this might end up in conflicts.

Preview:

![vivaldi_RWhh6zYYdg](https://user-images.githubusercontent.com/671982/181016733-76ab8d11-7b69-4413-aa79-1237e07fe708.png)

We'd love to hear your thoughts on this one and discuss the best approach. We're happy to further implement this feature with your guidance to make sure we're using the right approach.

Thanks!

Ramon

**Comment by benhaynes** [comment]
I think handling this for the translation interface is a great start to broader RTL support.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
